modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase

```diff
@@ -1675,6 +1675,132 @@ definition {
+				objectName = "CustomObject185",
+				value = "Entry Test");
+		}
+
+		task ("And: An new user is added") {
+			JSONUser.addUser(
+				userEmailAddress = "userea@liferay.com",
+				userFirstName = "userfn",
+				userLastName = "userln",
+				userScreenName = "usersn");
+
+			JSONUser.setFirstPassword(
+				agreeToTermsAndAnswerReminderQuery = "true",
+				requireReset = "false",
+				userEmailAddress = "userea@liferay.com");
+		}
+
+		task ("When: A new role is added and given permissions to view the Custom Object and trigger the standalone action via API and UI") {
+			JSONRole.addRegularRole(roleTitle = "CanSeeStandaloneActionWithPermission Role");
+
+			Role.openRolesAdmin();
+
+			LexiconEntry.gotoEntry(rowEntry = "CanSeeStandaloneActionWithPermission Role");
+
+			Navigator.gotoNavItem(navItem = "Define Permissions");
+
+			ObjectAdmin.searchInDefinePermissions(search = "Custom Objects 185");
+
+			Click(
+				key_wikiPageVersionNew = "action.ActionName",
+				locator1 = "WikiTable#SELECT_VERSION_NEW_VERSION_CHECKBOX");
+
+			PortletEntry.save();
+
+			var objectId = JSONObject.getObjectId(objectName = "CustomObject185");
+
+			Permissions.definePermissionViaJSONAPI(
+				resourceAction = "VIEW_CONTROL_PANEL",
+				resourceName = 90,
+				roleTitle = "CanSeeStandaloneActionWithPermission Role",
+				roleType = "regular");
+
+			Permissions.definePermissionViaJSONAPI(
+				resourceAction = "ACCESS_IN_CONTROL_PANEL",
+				resourceName = "com_liferay_object_web_internal_object_definitions_portlet_ObjectDefinitionsPortlet_${objectId}",
+				roleTitle = "CanSeeStandaloneActionWithPermission Role",
+				roleType = "regular");
+
+			Permissions.definePermissionViaJSONAPI(
+				resourceAction = "VIEW",
+				resourceName = "com_liferay_object_web_internal_object_definitions_portlet_ObjectDefinitionsPortlet_${objectId}",
+				roleTitle = "CanSeeStandaloneActionWithPermission Role",
+				roleType = "regular");
+
+			Permissions.definePermissionViaJSONAPI(
+				resourceAction = "VIEW",
+				resourceName = "com.liferay.object.model.ObjectDefinition#${objectId}",
+				roleTitle = "CanSeeStandaloneActionWithPermission Role",
+				roleType = "regular");
+		}
+
+		task ("Then: New user is able to trigger the action in the drop-down list") {
+			JSONRole.assignRoleToUser(
+				roleTitle = "CanSeeStandaloneActionWithPermission Role",
+				userEmailAddress = "userea@liferay.com");
+
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "userea@liferay.com",
+				userLoginFullName = "userfn userln");
+
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject185");
+
+			Click(
+				key_entry = "Entry Test",
+				locator1 = "ObjectPortlet#ENTRY_KEBAB_MENU");
+
+			Click(
+				key_dropdowntItem = "Action Label",
+				locator1 = "ObjectPortlet#CHECK_UNCHECK_DROPDOWN_ITEM");
+
+			AssertTextEquals(
+				key_entryDetail = "userfn userln",
+				key_settingHeader = "",
+				locator1 = "Account#DEFAULT_SETTINGS_TABLE_ITEM",
+				value1 = ${key_entryDetail});
```

Comment:
Here you assert the entry created by the User but is more recommendable that you assert the message generated to Action.

`Ex: Action entry`